specs-go/state.go

```diff
@@ -13,5 +13,5 @@ type State struct {
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
 	// Annotations are the annotations associated with the container.
-	Annotations map[string]string `json:"annotations"`
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Adding `omitempty` (as you do here and as #507 does not do) makes sense.  Adding REQUIRED/OPTIONAL to the Markdown spec (as #507 does and you do not do here) also makes sense.  Once both PRs land, we'll be in the right place ;).